### ISSUE TITLE
Selectable music for Blitz

### ISSUE THREAD

**Issue by mat1jaczyyy** [tetrio/issues#14]
It's currently impossible to pick music for Blitz, allow for picking from the soundtrack like 40 Line.

**Comment by LayneTheEpic** [comment]
Still relevant o.o

**Comment by mat1jaczyyy** [comment]
a

**Comment by FishPyramid** [comment]
I'm tired of listening to hyper velocity :(

**Comment by ZaptorZap** [comment]
yeah this is still a thing people want

**Comment by flashwave** [comment]
I remember pointing this out to osk back in early 2020 and he said that it was because the track in use is exactly the length of a Blitz game.

**Comment by ZaptorZap** [comment]
I'm well aware why Hyper Velocity is the default song, but it's pretty common knowledge that _at this point, with numerous retries_ for ideal RNG, particularly after Stride Mode was released, **no Blitz session from any competent player lasts 2 minutes long.** Here's a small sampling of some users I could catch Discord rich presence from after a bit of scrolling through member lists:
![image](https://user-images.githubusercontent.com/64891705/163089757-2242d158-b4ad-402d-bc39-07ef04166b96.png)

Furthermore, when this request was made, I believe this functionality/toggle didn't exist:
![image](https://user-images.githubusercontent.com/64891705/163089988-b8dd1a92-29f5-4721-9675-58353fad98a1.png)
If it didn't, then there'd really be no reason to implement this, but it does exist, so players wouldn't be subject to a song that forcefully restarts every time they retry in Blitz. This is one of the reasons I don't personally play a lot of Blitz, so I'm sure it's also waded off a lot of other users from playing it. Variety is the spice of life and all.

**Comment by mat1jaczyyy** [comment]
Do not reset music when retrying existing negates the reasoning that the current song is forced due to being 2 minutes long

**Comment by ZaptorZap** [comment]
Precisely, you hear the first four notes of Hyper Velocity and the first four notes exclusively for a good long while because of this. I'm saying other songs **wouldn't reset,**(but they would've when this request was made) so a Blitz grinder may find their retrying experience a bit more livelier with songs that actually progress.
